x-pack/plugins/observability_solution/dataset_quality/public/components/common/vertical_rule.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React from 'react';
+
+export const VerticalRule = (props: React.ComponentProps<'span'>) => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <span
+      css={css`
+        width: 1px;
```

Comment:
Suggestion: We can use EUI tokens for this 🙂 

```
width: {euiTheme.border.width.thin};
```

Reply:
Addressed with [style(logs): update style suggestions](https://github.com/elastic/kibana/pull/202746/commits/7b12855cc171332cadbb533ee98505855aa016b6)